disk/disk_freebsd.go

```diff
@@ -77,9 +77,9 @@ func Partitions(all bool) ([]PartitionStat, error) {
 		}
 
 		d := PartitionStat{
-			Device:     common.IntToString(stat.Mntfromname[:]),
-			Mountpoint: common.IntToString(stat.Mntonname[:]),
-			Fstype:     common.IntToString(stat.Fstypename[:]),
+			Device:     common.ByteToString(stat.Mntfromname[:]),
```

Comment:
on `386` and `amd64`, `Statfs.Mntfromname` is `[88]int8` so this cannot be `ByteToString`. I also don't see the generated types in `disk_freebsd_arm64.go`.

Reply:
ah, looks like upstream the type was changed to `unix.Statfs_t` which would work with your changes. Doesn't change that this PR won't build as-is.